bot/backend/apis/music.py

```diff
@@ -0,0 +1,48 @@
+"""interacting with the music api"""
+
+import tekore
+from loguru import logger
+from decouple import config
+from bot.backend.exceptions import ContentNotFoundError
```

Comment:
[PEP8 says that we should group imports into 3:](https://www.python.org/dev/peps/pep-0008/#imports)

- stdlib imports first
- 3rd party libs after it
- local package imports last
leaving a line between each group

```suggestion
import tekore
from decouple import config
from loguru import logger

from bot.backend.exceptions import ContentNotFoundError
```
Some formatting guides also tell us to sort imports alphabetically